Utilities/Utilities.cs

```diff
@@ -17,5 +19,12 @@ public static object GetAsync(object serObj, string url, object deserObj)
 
             return result;
         }
+
+        public static int GetCurrentUserId(ClaimsPrincipal user)
```

Comment:
Maybe use Common Utilities library instead of creating additional one for each service?